debian/changelog

```diff
@@ -1,3 +1,17 @@
+ubuntu-advantage-tools (27.6~22.04.1) jammy; urgency=medium
+
+  * d/tools.postinst:
+    - make log files world readable
+  * New upstream release 27.6 (LP: #1958556)
+    - cli: only go for resources on explicit help calls
+    - fips:
+      + allow enabling FIPS on focal clouds
+      + update prompt messages
+    - jobs: disable jobs after attach/auto-attach
```

Comment:
Specifically the license-check job
```suggestion
    - jobs: disable license-check job on GCP after attach
```

Reply:
Just modified the message a little bit for size constraints. Let me know what you think